com.sweethome3d.Desktop.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id" : "com.sweethome3d.Desktop",
+    "branch": "stable",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "19.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "SweetHome3D",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+         "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin",
+         "--env=JAVA_HOME=/app/jdk"
```

Comment:
Just to make sure, does the application ship its own JDK copy?

Reply:
It does. I should probably remove this, as this is already handled in the app's startup scripts.